### ISSUE TITLE
Foundational C++ Support macOS support claritication 

### ISSUE THREAD

**Issue by traversaro** [google/oss-policies-info#3]
Hello, thanks for sharing this repo. 

I was checking the Foundational C++ Support at https://github.com/google/oss-policies-info/blob/8067c719150dfec6a836dd82230c5eb0ba11acd7/foundational-cxx-support-matrix.md, and I noticed that for macOS : you have:
> `| macOS           | >= 10.1 (El Capitan) | 2022-07-01   |`

macOS 10.1 is quite old, but then I was that you indicate El Capitan, that is macOS 10.11 (see https://en.wikipedia.org/wiki/OS_X_El_Capitan), so I guess that the 10.1 is just a typo for 10.11, right?

Thanks in advance.

**Comment by devjgm** [comment]
Yes, that's a typo. I created https://github.com/google/oss-policies-info/pull/4 to fix it. Thanks for letting us know!

**Comment by traversaro** [comment]
Thanks for the fix!
